src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -315,13 +311,29 @@ DataView::ActionStatus LiveFunctionsDataView::GetActionStatus(
     return DataView::GetActionStatus(action, clicked_index, selected_indices);
   }
 
-  for (int index : selected_indices) {
-    uint64_t scope_id = GetScopeId(index);
-    const FunctionInfo& instrumented_function = *GetFunctionInfoFromRow(index);
-    if (is_visible_action_enabled(scope_id, instrumented_function)) {
-      return ActionStatus::kVisibleAndEnabled;
-    }
+  std::vector<const FunctionInfo*> function_infos;
+  std::transform(std::begin(selected_indices), std::end(selected_indices),
```

Comment:
As we don't wish to make the meaningless/non-emplemented entries invisible, the code is much simpler now. May I have a simple std::any_of instead of the loop?